test.sh

```diff
@@ -0,0 +1 @@
+pykern test
```

Comment:
You don't need test.sh. This is only for sirepo which runs some js stuff as well.

Reply:
I did it this way because if there isn't a `test.sh`, `python_ci_main()` will run `pykern ci run`, but I only want to run tests (no formatting etc). Is there another way to only run tests without having to update `radiasoft-download.sh` or `ci.py` in pykern?

Reply:
You did it right, sorry.